### ISSUE TITLE
intermittent test failures

### ISSUE THREAD

**Issue by ericsnowcurrently** [ericsnowcurrently/multi-core-python#13]
https://bugs.python.org/issue33868

**Comment by LewisGaul** [comment]
I notice the linked bpo issue has been moved to 'closed' with resolution 'out of date' (on 2019-09-20) - should this issue also be closed or is there still something to look into here?

**Comment by LewisGaul** [comment]
I now realise there is this open issue that this could refer to: https://bugs.python.org/issue37224.

**Comment by ericsnowcurrently** [comment]
Yeah, the linked issue isn't right.  In fact, there are several issues open.  I'll add the links.
